### PR TITLE
feat(aio): add footer links with GoogleFeedbackService

### PR DESCRIPTION
**Pull request by wardbell**
Added footer links and their styles

Incidentally folded `top-menu.component.scss` into the `styles/1-layouts/_top-menu.scss` per #15595

**CONFLICT with PR #15541** (just merge as I submitted). I did my best to merge. But something in the change to the footer background image generation prevents clicking on links!

**Disabled** that background temporarily  (see `_footer.scss` line 81) until someone (@sjtrimble?) can figure out how to make it work. Uncomment that line (`// content: "";`) to see the problem.

----
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: aio/src/app/shared/google-feedback.service.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  api: { startFeedback: (config: any) => void };
+}
+
+@Injectable()
+/**
+ * Google Feedback Service - opens a Google Feedback facility.
+ * Presupposes that tiny Google Feedback has been loaded from
+ * //www.gstatic.com/feedback/api.js with a script on the host web page.
+ */
+export class GoogleFeedbackService {
```
Comment (petebacondarwin) [inline aio/src/app/shared/google-feedback.service.ts]: Not `GfService`? :-P j/k

File: aio/src/index.html [hunk 2]
```diff
@@ -42,4 +32,17 @@
   <!-- TODO: google feedback -->
   <!-- TODO: twitter widget (but only on pages that use twitter) -->
 </body>
+
+<!-- Google Analytics -->
+<script>
```
Comment (IgorMinar) [inline aio/src/index.html]: this tag is outside of the `<body>` which makes it invalid html.

also I think it would be better to keep it in the header because we want the browse to start fetching this stuff early in the background. all these scripts are `async` which means that the browser will not block anything when fetching them.

**Review by IgorMinar — CHANGES_REQUESTED** [review]

File: aio/src/index.html (hunk 2 above)
Comment (wardbell) [inline aio/src/index.html]: Fixed

**Review by petebacondarwin — APPROVED** [review]

File: aio/src/styles/1-layouts/_footer.scss [hunk 3]
```diff
@@ -1,40 +1,91 @@
...
+  @media (max-width: 600px) {
+    h3 {
+      font-size: 100%;
     }
+  }
 }
 
 footer::after {
-    content: "";
+    // content: "";
```
Comment (IgorMinar) [inline aio/src/styles/1-layouts/_footer.scss]: please remove?

File: aio/src/index.html [hunk 4]
```diff
@@ -22,27 +22,20 @@
...
   window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-  // let GaService set the following
-  // ga('create', 'UA-xxxxxxx-1', 'auto');
-  // ga('send', 'pageview');
   </script>
   <script async src='https://www.google-analytics.com/analytics.js'></script>
   <!-- End Google Analytics -->
 
+  <!-- Google Feedback -->
+  <script async src="//www.gstatic.com/feedback/api.js" type="text/javascript"></script>
```
Comment (IgorMinar) [inline aio/src/index.html]: I thought Jules said no to google feedback. If you want to keep this work preserved in the repo history just in case we do want this feature then please add a commit to this PR that removes the GF and we won't squash it.

File: aio/content/license.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+The MIT License
```
Comment (IgorMinar) [inline aio/content/license.md]: why is this here? we removed all licenses from all repos and point to angular.io/license instead. do we need the license file here?

**Review by IgorMinar — CHANGES_REQUESTED** [review]

File: aio/content/license.md (hunk 5 above)
Comment (wardbell) [inline aio/content/license.md]: It's in the current site.

File: aio/src/index.html (hunk 4 above)
Comment (wardbell) [inline aio/src/index.html]: When did she say that? Last I heard she was asking Naomi. I'll remove it next PR

File: aio/src/styles/1-layouts/_footer.scss (hunk 3 above)
Comment (wardbell) [inline aio/src/styles/1-layouts/_footer.scss]: Can't. Doing so breaks. See comment above about how Stef has to fix the CSS. Please ... let's move on.

Comment (petebacondarwin) [inline aio/src/styles/1-layouts/_footer.scss]: I think he means remove the commented out line

Comment (wardbell) [inline aio/src/styles/1-layouts/_footer.scss]: Exactly. If I remove that commented out line, Kaboom! Leaving it so Stef has the rest of this CSS bit to work with ... it was hers.

File: aio/content/license.md (hunk 5 above)
Comment (wardbell) [inline aio/content/license.md]: I investigated what we do today ... and that is we serve the text file at `angular.io/license.txt` ... which is what this footer is doing, albeit ***in the doc viewer frame***. It would be a hack to make it show _outside that frame_ (as if an external page) because the server intercepts location change requests to the  same origin and serves them within the frame.

We do not have a mechanism for serving anything within `angular.io` as an external page. It that is important, we should add that capability in a separate PR.

File: aio/src/index.html (hunk 4 above)
Comment (IgorMinar) [inline aio/src/index.html]: the slack she said remove it.

File: aio/content/license.md (hunk 5 above)
Comment (IgorMinar) [inline aio/content/license.md]: ah. I get it. this is the file that is displayed when user navigates to angular.io/license. in that case we do need it :)

File: aio/src/index.html (hunk 4 above)
Comment (IgorMinar) [inline aio/src/index.html]: Please send a PR to remove it today.
